blocks/hero/hero.css

```diff
@@ -12,28 +12,85 @@ main .hero {
   position: relative;
   padding: 32px;
   min-height: 300px;
+  display: flex;
+  max-width: var(--max-content-width);
+  margin: auto;
+}
+
+main .hero .left-box {
+  background: #024c99;
```

Comment:
I believe this is the same as `--clr-blue` defined in styles.css, similarly below, the circle colors appear to be the same as `--clr-light-clue`